plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/navigator/actions/NavigatorHandlerRefresh.java

```diff
@@ -119,21 +120,25 @@ private boolean refreshInNavigator(ExecutionEvent event, IWorkbenchPart workbenc
             }
         }
         
-        return refreshInNavigator(this, refreshObjects);
+        return prepareRefresh(this, refreshObjects);
     }
     
-    public static boolean refreshInNavigator(Object source, List<DBNNode> refreshObjects) {
+    public static RefreshStatus refresh(Object source, List<? extends DBNNode> refreshObjects) {
```

Comment:
Please specify nullability annotations for the method's parameters and its return type.